.gitignore

```diff
@@ -9,4 +9,5 @@ data/
 dist*
 _build/
 test*.md
-run.sh
\ No newline at end of file
+run.sh
+.history/
```

Comment:
Could you remove this lane? (A bit weird for others lol)

Reply:
Could I contain .history/ for my plugins? Please! Thks.
run.sh is not my addition.

Reply:
Can I know what plugin are you using? Are you able to turn it off?
(Since other contributors also contribute to the repo, we may not wish everyone add their plug-in...)

Reply:
The plugin's name is LocalHistory and used in VSCode for save the local file history.
Should I commit the new version without the two lines?

Reply:
Yes, please keep the `.gitignore` file unchanged.

Reply:
I have searched the history of the .gitignore, and found run.sh is added by mli. So could you check it again before I commit the new version of .gitignore and tell me how to do with the file?

Reply:
Cool thanks for figuring out the root!